improver_tests/cube_combiner/test_CubeCombiner.py

```diff
@@ -184,6 +185,44 @@ def test_mean(self):
         self.assertEqual(result.name(), "new_cube_name")
         self.assertArrayAlmostEqual(result.data, expected_data)
 
+    def test_addition_cell_method_coordinate(self):
+        """Test that a cell method is not added, if a cell method coordinate is
+        provided and an addition operation is undertaken."""
+        plugin = CubeCombiner("add", cell_method_coordinate="time")
+        cubelist = iris.cube.CubeList([self.cube1, self.cube2])
+        input_copy = deepcopy(cubelist)
+        result = plugin.process(cubelist, "new_cube_name")
+        expected_data = np.full((2, 2), 1.1, dtype=np.float32)
+        self.assertArrayAlmostEqual(result.data, expected_data)
+        self.assertArrayAlmostEqual(result.cell_methods, ())
+        self.assertCubeListEqual(input_copy, cubelist)
```

Comment:
I'd say check that result.cell_methods is None, but as I want an exception, there's not much point.

Reply:
I've added an exception.